parlai/chat_service/services/websocket/websocket_manager.py

```diff
@@ -418,7 +191,7 @@ def _create_agent(self, task_id, socketID):
         :param agent_id:
             int agent id
         """
-        return WebsocketAgent(self.opt, self, task_id, socketID)
+        return WebsocketAgent(self.opt, self, socketID, task_id)
```

Comment:
ooo tricky that we swapped this

Reply:
Yeah I had it in the wrong order before the base class was created. Now it's consistent with the other agents